contexts/wagmi-provider.tsx

```diff
@@ -30,10 +30,21 @@ const coinEx = defineChain({
   }
 });
 
+const ganache = defineChain({
+  ...localhost,
+  blockExplorers: {
+    default: {
+      name: "Local",
+      url: "http://127.0.0.1:8545",
+    }
+  }
+});
+
 const config = getDefaultConfig({
     appName: "BEPRO",
     projectId: publicRuntimeConfig?.walletConnectProjectId || "bc2288336095f20ebf8653a1ab670566",
     chains: [
+      ganache,
```

Comment:
Isn't it better to ensure the ganache chain is only included when running in a development environment?
```ts
chains: [
  polygon,
  polygonAmoy,
  polygonMumbai,
  aurora,
  auroraTestnet,
  moonbeam,
  coinEx,
  mainnet,
  ...(!isProduction ? [ganache] : []),
],
```

Reply:
Yes, it could be a good way to avoid mistakes, but at the end, the application will always ask the user to change to the correct network.